fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -86,6 +86,7 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     if ctx.obj["CONFIG"].user.analytics_opt_out is False:  # requires explicit opt-in
         ctx.meta["ANALYTICS_CLIENT"] = AnalyticsClient(
             client_id=ctx.obj["CONFIG"].cli.analytics_id,
+            developer_mode=(getenv("FIDESCTL_TEST_MODE") == "True"),
```

Comment:
nit but do we want to keep explicitly adding `bool()` for these? Might be nice to keep those aligned either way 🤷🏽